site/src/theme/dark/roles.ts

```diff
@@ -143,13 +143,36 @@ export default {
 		},
 	},
 	preview: {
```

Comment:
Previously, the previews were only being used for the theme selector buttons, but with how the conditionals were set up, they wouldn't ever be able to trigger

Figured it'd be safe to update the theme in this case

Reply:
these colors really break from the pattern of the other roles. I think I'd rather they live outside of the roles paradigm.